packages/common-i18n/src/locales/sv/common.json

```
@@ -82,5 +82,7 @@
   },
   "share": {
     "article": "Share article"
-  }
+  },
+  "next": "Nästa",
+  "previous": "Tidigare"
```

MUST BE "föregående".

```suggestion
  "next": "Nästa",
  "previous": "Föregående"
```